build.xml

```diff
@@ -1,11 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project name="build-sample-connector" basedir="." default="all">
+<project name="build-jira-connector" basedir="." default="all">
+
+    <property name="BUNDLE_NAME" value="Connector-JIRA"/>
```

Comment:
You shouldn't define these properties in this file. They are defined in build.properties.

Please don't modify the build files structure, it's the same for all connectors.